### DISCUSSION TITLE
An erroe is found in Page/StackPanel/Grid

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13200] [Q&A]
The Output view show these errors:

Bookcase.xaml(11,6,11,6): error UXAML0001: An error was found in Grid
 
sometimes Grid,sometimes Page,or other controls.

And It will give a message:
Uno.UI.SourceGenerators.XamlGenerator.XamlParsingException: An error was found in Page ---> Uno.UI.SourceGenerators.XamlGenerator.XamlParsingException: An error was found in Grid ---> System.InvalidOperationException: Unable to find content value on Uno.UI.SourceGenerators.XamlGenerator.XamlObjectDefinition

But not specific where the error is .

The xaml file don't have bugs, I think.

I found a similar issue in https://github.com/unoplatform/uno/issues/12862 , but it is about .net 8, my project is .net 7. I tried follow the resolvement,but it doesn't work.

**Comment by limeniye** [comment]
@DJDQfff  Good day! 
Please, send the archive.

**Reply by jeromelaban** [reply]
Indeed, could you provide a binlog, following [these directions](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?
